_locales/en/messages.json

```diff
@@ -154,5 +154,32 @@
   },
   "search": {
     "message": "Search"
+  },
+  "permissionsTitle": {
+    "message": "Permissions - Scratch Addons"
+  },
+  "permissions": {
+    "message": "Permissions"
+  },
+  "enablePermissionsTitle": {
+    "message": "Please enable site access"
+  },
+  "enablePermissionsDescription": {
+    "message": "Scratch Addons needs site access permission to run addons."
```

Comment:
Too technical. What about `Scratch Addons needs site access permission to work.`?